BaseTools/SecurityFixes.yaml

```diff
@@ -0,0 +1,27 @@
+CVE_2017_5731__CVE_2017_5732__CVE_2017_5733__CVE_2017_5734__CVE_2017_5735:
+    commit_titles:
+        - # Upstream Patch from Tianocore Edk2
+            - "BaseTools: Add more checker in Decompress algorithm to access the valid buffer (CVE FIX)"
+    cve:
+        - CVE-2017-5731 # This is the only CVE that still is in use
```

Comment:
discard the abandoned CVE's and change the name,
This could just be a note field